octobot_trading/personal_data/orders/states/close_order_state.py

```diff
@@ -40,6 +40,9 @@ async def on_refresh_successful(self):
         """
         Verify the order is properly closed
         """
+        if self.order is None:
```

Comment:
Shouldn't we check if the order is cleared?

Reply:
updated